src/Gui/DlgSettingsWorkbenchesImp.cpp

```diff
@@ -0,0 +1,475 @@
+/***************************************************************************
+ *   Copyright (c) 2020 Chris Hennes (chennes@pioneerlibrarysystem.org)    *
+ *                                                                         *
+ *   This file is part of the FreeCAD CAx development system.              *
+ *                                                                         *
+ *   This library is free software; you can redistribute it and/or         *
+ *   modify it under the terms of the GNU Library General Public           *
+ *   License as published by the Free Software Foundation; either          *
+ *   version 2 of the License, or (at your option) any later version.      *
+ *                                                                         *
+ *   This library  is distributed in the hope that it will be useful,      *
+ *   but WITHOUT ANY WARRANTY; without even the implied warranty of        *
+ *   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the         *
+ *   GNU Library General Public License for more details.                  *
+ *                                                                         *
+ *   You should have received a copy of the GNU Library General Public     *
+ *   License along with this library; see the file COPYING.LIB. If not,    *
+ *   write to the Free Software Foundation, Inc., 59 Temple Place,         *
+ *   Suite 330, Boston, MA  02111-1307, USA                                *
+ *                                                                         *
+ ***************************************************************************/
+
+#include "PreCompiled.h"
+#ifndef _PreComp_
+#include <QCheckBox>
+#include <QPushButton>
+#include <sstream>
+#endif
+
+#include "DlgSettingsWorkbenchesImp.h"
+#include "ui_DlgSettingsWorkbenches.h"
+#include "Application.h"
+#include "UserSettings.h"
+#include "Workbench.h"
+#include "WorkbenchManager.h"
+
+
+using namespace Gui::Dialog;
+
+const QString DlgSettingsWorkbenchesImp::iconLabelStr = QString::fromLatin1("iconLabel");
+const QString DlgSettingsWorkbenchesImp::nameLabelStr = QString::fromLatin1("nameLabel");
+const QString DlgSettingsWorkbenchesImp::loadLabelStr = QString::fromLatin1("loadLabel");
+const QString DlgSettingsWorkbenchesImp::loadButtonStr = QString::fromLatin1("loadButton");
+const QString DlgSettingsWorkbenchesImp::enableCheckboxStr = QString::fromLatin1("enableCheckbox");
+const QString DlgSettingsWorkbenchesImp::autoloadCheckboxStr = QString::fromLatin1("autoloadCheckbox");
+
+/* TRANSLATOR Gui::Dialog::DlgSettingsWorkbenchesImp */
+
+/**
+ *  Constructs a DlgSettingsWorkbenchesImp
+ */
+DlgSettingsWorkbenchesImp::DlgSettingsWorkbenchesImp( QWidget* parent )
+    : PreferencePage( parent )
+    , ui(new Ui_DlgSettingsWorkbenches)
+{
+    ui->setupUi(this);
+    connect(ui->AutoloadModuleCombo, QOverload<int>::of(&QComboBox::activated), this, [this](int index) { onStartWbChangedClicked(index); });
```

Comment:
Is using `qOverload` still a problem with MSVC ?
```suggestion
    connect(ui->AutoloadModuleCombo, qOverload<int>(&QComboBox::activated), this, [this](int index) { onStartWbChangedClicked(index); });
```

Reply:
Without qOverload or with your version it doesn't build on MSVC

Reply:
OK, I remembered MSVC generated issues with this...
It's solved from MSVC 2019 IIRC.